src/infra/api/oi.tpl.json

```diff
@@ -223,6 +223,46 @@
         }
       }
     },
+    "/login-alert-message.json": {
+      "get": {
+        "tags": [
+          "Login alert message information"
+        ],
+        "description": "This route will serve the login alert messages",
+        "parameters": [],
+        "x-amazon-apigateway-integration": {
+          "credentials" : "${s3_apigateway_proxy_role}",
+          "httpMethod" : "GET",
+          "uri" : "${assets_bucket_uri}/login-alert-message.json",
+          "type" : "aws",          
+          "passthroughBehavior": "WHEN_NO_TEMPLATES",
+          "requestParameters": {
```

Comment:
Is this useful? I mean: the json file has no logic capable to interpret any query string parameters  does it?

Reply:
no, it's not. You're right, we don't definitely need those parameters. Thanks